### PR TITLE
Add new autocomplete and add tags settings VC-1568 VC-1552

### PR DESCRIPTION
**Pull request by konutis**

### CODE REVIEW

File: visualcomposer/Modules/Editors/Settings/WordPressSettings/TagsController.php [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+class TagsController extends Container implements Module
+{
+    use WpFiltersActions;
+    use EventsFilters;
+
+    public function __construct()
+    {
+        $this->addFilter(
+            'vcv:ajax:autocomplete:findTag:adminNonce',
```
Comment (pavelthq) [inline visualcomposer/Modules/Editors/Settings/WordPressSettings/TagsController.php:26]: please change namespace to more specific like
`vcv:ajax:editors:settings:tags:autocomplete:findTag:adminNonce`

File: visualcomposer/Modules/Editors/Settings/WordPressSettings/TagsController.php [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        PostType $postTypeHelper
+    ) {
+        $currentPost = $postTypeHelper->get();
+        $postTypeTaxonomies = get_object_taxonomies($currentPost);
+        if (in_array('post_tag', $postTypeTaxonomies)) {
+            $tagTaxonomy = get_taxonomy('post_tag');
+            // @codingStandardsIgnoreLine
+            if (isset($currentPost->post_type) && ($tagTaxonomy->show_ui || false !== $tagTaxonomy->meta_box_cb)) {
+                $tags = get_the_tags($currentPost->ID);
+                $response = array_merge(
```
Comment (pavelthq) [inline visualcomposer/Modules/Editors/Settings/WordPressSettings/TagsController.php:60]: Change sturcture to $variables[] it is more efficient

File: visualcomposer/Modules/Editors/Settings/WordPressSettings/TagsController.php [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+use VisualComposer\Framework\Container;
+use VisualComposer\Framework\Illuminate\Support\Module;
+use VisualComposer\Helpers\PostType;
+use VisualComposer\Helpers\Request;
+use VisualComposer\Helpers\Traits\EventsFilters;
+use VisualComposer\Helpers\Traits\WpFiltersActions;
+
+class TagsController extends Container implements Module
+{
+    use WpFiltersActions;
```
Comment (pavelthq) [inline visualcomposer/Modules/Editors/Settings/WordPressSettings/TagsController.php:20]: you don't use WpFilterActions there, no need to add it

**Review by pavelthq — CHANGES_REQUESTED** [review]

File: public/sources/attributes/autocomplete/Component.js [hunk 4]
```diff
@@ -4,6 +4,8 @@ import Attribute from '../attribute'
 import TokenizationList from './lib/tokenizationList'
 import PropTypes from 'prop-types'
 
+import ReactTagContainer from './libNew/ReactTagContainer'
```
Comment (nikitahl) [inline public/sources/attributes/autocomplete/Component.js:7]: Maybe rename `libNew` to something more meaningful like `reactTagsLib` or something like that?

**Review by pavelthq — APPROVED** [review]

**Pull request merged**